extension/resolve-copied-redirect.js

```diff
@@ -0,0 +1,23 @@
+/* global
+
+  chrome
+*/
+'use strict';
+
+let getLinkElement = el => {
+  do {
+    if(el.href && el.href.startsWith('http')) {
+      return el;
+    }
+    el = el.parentNode;
+  } while(el !== null);
+};
+
+window.addEventListener('contextmenu', e => {
```

Comment:
Theoretically this causes a race condition with the contextmenu listener responsible of setting the tracking url. In my initial testing this did not appear to be a problem. 

Might be more effective to run the method in a later cycle with `setTimeout(..., 0)`